packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -293,12 +295,119 @@ The following is an example System Log rate limit event where too many OTP verif
+                "state": null,
+                "country": null,
+                "postalCode": null,
+                "geolocation": {
+                    "lat": 36.62,
+                    "lon": -113.17
+                }
+            }
+        },
+        "authenticationContext": {
+            "authenticationProvider": null,
+            "credentialProvider": null,
+            "credentialType": null,
+            "issuer": null,
+            "interface": null,
+            "authenticationStep": 0,
+            "externalSessionId": "idx3GDqJrIfQhCRJTEuNC-l9A"
+        },
+        "displayMessage": "Operation rate limit violation",
+        "eventType": "system.operation.rate_limit.violation",
+        "outcome": {
+            "result": "DENY",
+            "reason": "Profile reload for john.smith@example.com skipped due to rate limiting"
+        },
+        "published": "2021-05-14T20:42:21.480Z",
+        "securityContext": {
+            "asNumber": null,
+            "asOrg": null,
+            "isp": null,
+            "domain": null,
+            "isProxy": null
+        },
+        "severity": "WARN",
+        "debugContext": {
+            "debugData": {
+                "operationRateLimitTimeUnit": "MINUTES",
+                "operationRateLimitScopeType": "user",
+                "deviceFingerprint": "3857b18395b101c0703feec226def544",
+                "operationRateLimitSecondsToReset": "96",
+                "requestId": "reqmXZkOgXpQdms4-3a8eD9mg",
+                "operationRateLimitThreshold": "1",
+                "operationRateLimitTimeSpan": "5",
+                "requestUri": "/idp/idx/identify",
+                "operationRateLimitType": "profile_reload",
+                "operationRateLimitSubtype": "AD agent",
+                "url": "/idp/idx/identify?"
+            }
+        },
+        "legacyEventType": null,
+        "transaction": {
+            "type": "WEB",
+            "id": "reqmXZkOgXpQdms4-3a8eD9mg",
+            "detail": {}
+        },
+        "uuid": "e19832a2-b4f4-11eb-9f1e-bba1874b8f01",
+        "version": "0",
+        "request": {
+            "ipChain": [
+                {
+                    "ip": "127.0.0.1",
+                    "geographicalContext": null,
+                    "version": "V4",
+                    "source": null
+                }
+            ]
+        },
+        "target": [
+            {
+                "id": "00u1ngpFSRLFie7vT0g4",
+                "type": "User",
+                "alternateId": "john.smith@example.com",
+                "displayName": "John Smith",
+                "detailEntry": null
+            }
+        ]
+    }
+```
+
 #### Operation rate limit subtypes
 
 The following table includes the available `Subtypes` for operation rate limits.
 
+> **Note:** The `AD agent` and `LDAP agent` are only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
```

Comment:
@chuanbozhang-okta can you expand on this... "`profile reload throttling` for the AD and LDAP agent are only available as a part of the Okta Identity Engine"

Reply:
As the comment currently stands, it would seem that AD and LDAP agent are not supported in V1 which is not true 😄

Reply:
maybe I shall say: "The `AD agent` and `LDAP agent`  subtypes"